library/badges.js

```diff
@@ -27,8 +27,15 @@ function badgesForUser (userInfo, user, hasLongSession) {
         badges += ':christmas_tree: [Practising knows no holiday](http://euge.ca/61)\n'
       }
 
+      let fishBadgeAwarded = false
+      if (user.rooms_practiced.includes('🐟')) {
+        badges += ':fish: Never gonna give 魚 up\n'
```

Comment:
Also assign`fishBadgeAwarded = true` here?

Reply:
Wow. That's hilarious.